themes/engineering/lib/scss/print/_news-print.scss

```diff
@@ -0,0 +1,149 @@
+//Removing unnecessary content
+.more-link,
+#main-content,
+h2.stanford-news__title,
+h2.block__title,
+.stanford-news--cards{
+  display: none;
+}
+
+.soe-print__header {
+  width: 25%;
+  font-size: 24px;
+
+  img.img-responsive {
+    width: 30%;
+    height: auto;
+  }
+}
+
+.soe-printable__page{
+  margin: 32px;
+}
+
+
+// Wrapping all styles in this class for added insurance against these being fired on other pages.
+// Not that this css file should be available anywhere else.
+.node--layout-printable{
+
+  //Typography
+  font-family: "Source Sans Pro", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  width: 100%;
+  font-size: 20px;
+
+  h1,
+  h2,
+  h3{
+    font-family: "Roboto Slab", serif;
+  }
+
+  h1{
+    font-size: 55px;
+    margin: 55px 0 24px 0;
+  }
+
+  //Header styles
+  .section-header {
+    margin-bottom: 20px;
+  }
+
+  .su-news-dek{
+    width: 554px;
+    margin-bottom: 12px;
+  }
+
+  .stanford-news.datetime{
+    &:after {
+      content: "\00a0\00a0";
+    }
+  }
+
+  .byline-social-media-wrapper {
+    margin: 0 0 12px;
+
+    .news-date-pipe-byline {
+      font-size: 18px;
+      display: flex;
+    }
+
+    .su-news-byline.node.stanford-news.string.label-hidden {
+      &:before {
+        content: '\00a0\00a0';
+      }
+    }
+  }
+
+  // Image and body styles
+  .su-news-banner-media-caption{
+    font-size: 16px;
+    color: #53565a;
+  }
+
+  .su-wysiwyg-text {
+    margin-top: 82px;
+  }
+}
+
+
+
+
+// @todo Remove the following, which was taken from the D7 SOE site, and I used as a reference.
```

Comment:
Let's go ahead and remove this now.  It's in the commit history, so we can find it again if necessary.

Reply:
Dones